spec/controllers/games_controller_spec.rb

```diff
@@ -58,5 +58,17 @@
     end  
   end
 
+  describe "games#update action"  do
+    it "should allow users to join a game as player 2" do
+      user = FactoryGirl.create(:user)
+      sign_in user
+      g = FactoryGirl.create(:game, sith_user_id: user.id) 
+      g2 = FactoryGirl.create(:game, jedi_user_id: user.id)
```

Comment:
This is setting up someone to be both sith_user AND jedi_user.  I think we were talking about validations that this isn't possible.